unlock-app/src/__tests__/components/interface/GlobalErrorConsumer.test.js

```diff
@@ -1,28 +1,29 @@
 import React from 'react'
 import * as rtl from 'react-testing-library'
-
+import { Provider } from 'react-redux'
 import GlobalErrorConsumer, {
   displayError,
 } from '../../../components/interface/GlobalErrorConsumer'
 import { GlobalErrorContext } from '../../../utils/GlobalErrorProvider'
 import { FATAL_NO_USER_ACCOUNT, FATAL_MISSING_PROVIDER } from '../../../errors'
+import createUnlockStore from '../../../createUnlockStore'
 
-const Provider = GlobalErrorContext.Provider
+const ErrorProvider = GlobalErrorContext.Provider
```

Comment:
I am a bit confused as to why these changes need to be in this PR?

Reply:
No problem at all, I will get started on splitting this!